modules/main-rules/pages/transportation.adoc

```diff
@@ -1,59 +1,62 @@
+To refer to the units that we transport, we sometimes use the term *cargo*.
 
-== Transportation by {transport-x} units
+[NOTE.e40k]
+.Transportation
+====
+The Transport rules in _Epic 40,000_ leave a lot of room for interpretation, and they aren't entirely clear on some of the commonalities and differences between _Transport_ and _Hitch_.
+Nor do they account well for disparities that stem from the range of different base sizes that Epic enthusiasts use for their units these days.
 
-Units that have the {transport-x} ability can carry other units across the battlefield.
-They can only transport Infantry units -- except where the rules or army lists say otherwise.
+To minimise these problems, we've integrated the {transport-x} and {hitch-x} abilities with some new conditions.
+====
 
-To embark or disembark a unit:
+NOTE: To be clear, a transportation unit does not gain the abilities of any embarked units.
 
-* You must sacrifice 5 cm from the movement of the {transport-x} unit.
-* You must sacrifice 5 cm from the movement of the unit that you embark or disembark.
-* You may embark and disembark units in the same turn, the same phase or even in the same move -- so long as all of the involved units have enough movement.
+== Transportation by {transport-x} units
 
-You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example.
-Obviously, once you disembark them then the normal rules apply again.
+:by-transport-x:
+include::partial$transportation-by-transport-units.adoc[]
 
-[NOTE]
-.Placement of disembarked units
-====
-When you disembark units, measure their movement from the base of the {transport-x} unit.
-If you do not have enough movement to clear the {transport-x} unit's base then place the disembarked unit in base contact with the {transport-x} unit.
-====
+:!by-transport-x:
 
-[NOTE.e40k]
-.Placement of disembarked units
-====
-We added this rule to minimise the effects that result from the assortment of different base sizes that players use.
-====
+== Transportation by {hitch-x} units
 
-== Destruction of {transport-x} units
+:by-hitch-x:
+include::partial$transportation-by-transport-units.adoc[]
 
-If a {transport-x} unit is destroyed then, just before you remove the unit as a casualty, roll a D6 for each embarked unit: On a result of 1, 2 or 3, the unit suffers a hit.
-If the unit survives, place it in contact with the destroyed {transport-x} unit.
+:!by-hitch-x:
 
-Then remove the {transport-x} unit as normal.
+== Destruction of transportation units
 
-== Transportation by {hitch-x} units
+If a transportation unit is destroyed while cargo units are embarked then, before you remove the unit as a casualty, roll a D6 for each embarked unit: On a result of 1, 2 or 3, the embarked unit suffers a hit.
+If the embarked unit survives, place it completely within 5 cm of the destroyed transportation unit.
```

Comment:
Should this now refer to cargo units throughout? for example:

```suggestion
If a transportation unit is destroyed while cargo units are embarked then, before you remove the unit as a casualty, roll a D6 for each cargo unit: On a result of 1, 2 or 3, the cargo unit suffers a hit.
If the cargo unit survives, place it completely within 5 cm of the destroyed transportation unit.
```

Reply:
I started to do exactly that, and then undid it. It would be intelligible either way, and I wasn't 100% sure which is best. But I figured that it is the specific 'embarked' state of the cargo that is most pertinent here.

---

As an aside on the new term:

I added the term 'cargo' in *some* places because there are sentences where it helps to clearly disambiguate the cargo units from other units, especially but not only the transport units.

Part of the trouble is that while 'embarked units' or 'disembarked units' work in many cases, 'unembarked unit' doesn't really seem to fit. And of course 'units that you intend to embark in a moment, but haven't embarked just yet' isn't great either :-D Whereas the term 'cargo unit' seems to work in those cases. We don't really need 'cargo' where 'embarked unit' or 'disembarked unit' cover the situation. And I don't think we really need 'embarked cargo unit' because this is implicit — embarked and disembarked states are relevant only for cargo units.

---

However, we could rationalise it some more, perhaps like this (also breaks down the long sentence that I should've caught before):
```suggestion
If a transportation unit is destroyed then any cargo unit that is embarked at the time may also suffer a hit.
Before you remove the destroyed transport unit, roll a D6 for each embarked unit: On a result of 1, 2 or 3, it suffers a hit.
If the unit survives, place it completely within 5 cm of the destroyed transportation unit.
```

Reply:
Yeah, that was a fairly minor suggestion anyway.

But I agree, the revised version helps break it up a bit more!